packages/client/components/header/index.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from 'react'
+import KeyboardArrowLeftIcon from '@material-ui/icons/KeyboardArrowLeft'
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown'
+import styled from 'styled-components'
+import ClickAwayListener from '@material-ui/core/ClickAwayListener'
+import Popper from '@material-ui/core/Popper'
+import Button from '@material-ui/core/Button'
+import Dialog from '@material-ui/core/Dialog'
+import DialogActions from '@material-ui/core/DialogActions'
+import DialogTitle from '@material-ui/core/DialogTitle'
+import { Logo } from '../icons/logo'
+
+const Container = styled.div`
+  height: 48px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 4px;
+  box-sizing: border-box;
+`
+
+const UserNameContainer = styled.div`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`
+
+const BackArrowContainer = styled.div`
+  height: 48px;
+  width: 48px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-left: 2px;
+`
+
+const LogoContainer = styled.div`
+  width: 48px;
+  height: 48px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`
+
+const Title = styled.div`
+  font-size: 16px;
+  position: absolute;
+  margin-left: 43%;
```

Comment:
> Are you sure this centers title? It looks like it's very dependent on the title length.
> Have you considered using a flexbox here with `space-between`? https://material-ui.com/components/grid/#interactive

I tried to use `space-between`. The title actually looks bad when it's centered using flex box. Because the icon/back button on the left is usually small comparing to user's name on the right.
![image](https://user-images.githubusercontent.com/16532416/70289349-75fee000-17a2-11ea-827d-6ff292f082c9.png)

Reply:
Alright. Also, we'll need to be careful about displaying the name there; they can be extremely long.